docs/content/examples/use-cases/ios-mail-push-support.md

```diff
@@ -0,0 +1,232 @@
+            # url:              protocol, hostname and port under which xapsd listens
+            # user_lookup: Use if you want to determine the username used for PNs from environment variables provided by
+            #                   login mechanism. Value is variable name to look up.
+            # max_retries:      maximum num of retries the http client connects to the xaps daemon
+            # timeout_msecs     http timeout of the http connection
+            xaps_config = url=http://127.0.0.1:11619 user_lookup=theattribute max_retries=6 timeout_msecs=5000
+            push_notification_driver = xaps
+        }
+        ```
+
+    - Create a supervisord file named `xapsd.conf` in `docker-data/dms/config/xaps` with the following content:
+        ```txt title="xapsd.conf"
+
+        [program:xapsd]
+        startsecs=0
+        autostart=false
+        autorestart=true
+        stdout_logfile=/var/log/supervisor/%(program_name)s.log
+        stderr_logfile=/var/log/supervisor/%(program_name)s.log
+        user=xapsd
+        command=/usr/bin/xapsd
+        pidfile=/var/run/xapsd/xapsd.pid
+        ```
+
+    - Create or update your `user-patches.sh` in `docker-data/dms/config` to move the files to their final location as well as starting the daemon service:
+        ```sh title="user-patches.sh"
+        #!/bin/bash
+
+        # Copy the configs to internal locations:
+        cp /tmp/docker-mailserver/xaps/95-xaps.conf /etc/dovecot/conf.d/95-xaps.conf
+        cp /tmp/docker-mailserver/xaps/xapsd.yaml /etc/xapsd/xapsd.yaml
+        cp /tmp/docker-mailserver/xaps/xapsd.conf /etc/supervisor/conf.d/xapsd.conf
+
+        # Setup data persistence and ensure ownership is always for xapsd:
+        mkdir -p /var/mail-state/lib-xapsd
+        chown -R xapsd:xapsd /var/mail-state/lib-xapsd
+
+        # Start the xaps daemon:
+        supervisorctl update
+        supervisorctl start xapsd
+        ```
+
+7. Recreate the container again to apply the new configuration:
+    ```sh
+    docker compose down
+    docker compose up -d
+    ```
+
+8. Recreate your mail account on your iOS device and check the logs in `/var/log/supervisor/dovecot.log` and `/var/log/supervisor/xapsd.log` for any errors.
+
+
+## Other configuration options
+
+Both device registration and notifications send a username to the daemon to lookup the device. While the registration and other IMAP operations in Dovecot will send the Dovecot username, LMTP will send the provided authentication username.
+
+The format of that username is specified by the `auth_username_format` Dovecot setting. If you are not using mail addresses as Dovecot usernames - e.g. when using LDAP - you can either change the `auth_username_format` or add the mail address as property to the user account and use the lookup feature (see below).
+
+```sh title="user-patches.sh"
+sed -i -r "s|^#?(auth_username_format =).*|\1 %Ln|" /etc/dovecot/conf.d/10-auth.conf
+```
```

Comment:
This was something I encountered recently while going through LDAP support in DMS, nice to have some proper docs in DMS for that! ❤️